src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.util.ts

```diff
@@ -386,4 +387,22 @@ export const selectStyles = {
         ...base,
         padding: '0 8px',
     }),
-}
\ No newline at end of file
+}
+
+/**
+ * @description This function is used to trim the manifest data by removing the managed fields from the manifest data
+ */
+export const getTrimmedManifestData = (
+    manifestData: ManifestData,
+    returnAsString: boolean = false,
+): ManifestData | string => {
+    if (manifestData[MANIFEST_KEY_FIELDS.METADATA]) {
+        const { [MANIFEST_KEY_FIELDS.MANAGED_FIELDS]: _, ...metadata } = manifestData[MANIFEST_KEY_FIELDS.METADATA]
```

Comment:
Can we consider using the delete operator?

Reply:
Using delete would change the existing object impacting the purity of function